src/gtkui/gtkui.h

```diff
@@ -37,6 +37,7 @@ extern Playlist menu_tab_playlist;
 extern const PluginPreferences gtkui_prefs;
 
 /* ui_gtk.c */
+GtkWindow * get_main_window ();
```

Comment:
What would you prefer? Getter function like this or `extern GtkWidget * window;` in `gtkui.h`?

Reply:
This is fine :)